### DISCUSSION TITLE
RimWorld Italian Steam Store Translation

### DISCUSSION THREAD

**Discussion by TiaPixel** [Ludeon/RimWorld-it#280] [General]
Hello, I'm Tia! I'm helping Ludeon Studios with translating the Royalty and Ideology Steam store pages. Would anyone from the GitHub Italian team be interested in helping me translate the store pages from English to Italian? :) 

Please respond with a way I can contact you, ideally Discord or email!

**Comment by Rexlord** [comment]
Lagorn  => Lagorn#4266

**Reply by TiaPixel** [reply]
> Lagorn => Lagorn#4266

I sent you a friend request!
